policy/modules/apps/mono.if

```diff
@@ -63,7 +63,7 @@ template(`mono_role_template',`
 	optional_policy(`
 		fs_dontaudit_rw_tmpfs_files($1_mono_t)
 
-		xserver_role($1_r, $1_mono_t)
+		xserver_role($1, $1_mono_t, $1_application_exec_domain, $1_r)
```

Comment:
This needs the same changes you  made elsewhere, instead of using `$1_application_exec_domain` directly.